spec/50-privacy.md

```diff
@@ -1,21 +1,23 @@
+Trace vendors MUST assess the risk of header abuse. This section provides some considerations and initial assessment of the risk associated with storing and propagating these headers. Trace vendors may choose to inspect and remove sensitive information from the fields before allowing the tracing system to execute code that potentially can propagate or store these fields. All mutations should, however, conform to the list of mutations defined in this specification.
 
 ## Privacy of traceparent field
 
-`traceparent` field has a predefined set of values. These values are randomly-generated numbers. If a random number generator has any logic of using user-identifiable information like IP address - this information may be exposed. Random number generators MUST NOT rely on any information that can potentially be user-identifiable.
+The `traceparent` field has a predefined set of values. These values are randomly-generated numbers. If a random number generator has any logic of using user-identifiable information like IP address - this information may be exposed. Random number generators MUST NOT rely on any information that can potentially be user-identifiable.
 
-Another privacy risk of the `traceparent` field is an ability to correlate calls made as part of a single transaction. A downstream service may track and correlate two or more calls made in a single transaction and make assumptions about the identity of the caller of one call base on information in another call. Service initiating calls MAY choose to restart trace while making calls that might identify caller in the downstream service.
+Another privacy risk of the `traceparent` field is an ability to correlate calls made as part of a single transaction. A downstream service may track and correlate two or more calls made in a single transaction and make assumptions about the identity of the caller of one call base on information in another call. Service initiating calls MAY choose to restart a trace while making calls that might identify caller in the downstream service.
 
 Note, both privacy concerns of `traceparent` field are theoretical rather than practical.
 
 ## Privacy of tracestate field
 
-The field `tracestate` may contain any opaque value in any of the keys. The main purpose of this header is to provide additional information about the position of request in the multiple distributed tracing graphs.
+The  `tracestate` field may contain any opaque value in any of the keys. The main purpose of this header is to provide additional vendor-specific trace-identification information across different distributed tracing systems. 
```

Comment:
I agree that initial sentence is not very clear. I think it should be changed to highlight the fact that this is positioning information, i.e. mostly random and meaningless identifiers rather than PII. Why do you think this section should highlight that thus us vendor specific information?

Reply:
Vendors can choose to put whatever they need in trace-state and not only positioning information (e.g. tenant information, sampling, ....)